tests/models/test_instrumented.py

```diff
@@ -692,3 +692,24 @@ def __repr__(self):
             'event.name': 'gen_ai.tool.message',
         },
     ]
+
+
+def test_messages_to_otel_events_instructions():
+    messages = [
+        ModelRequest(instructions='system_prompt', parts=[UserPromptPart('user_prompt')]),
+        ModelResponse(parts=[TextPart('text1')]),
+    ]
+    assert [
+        InstrumentedModel.event_to_dict(e) for e in InstrumentedModel.messages_to_otel_events(messages)
+    ] == snapshot(
+        [
+            {'content': 'system_prompt', 'role': 'system', 'event.name': 'gen_ai.system.message'},
```

Comment:
It doesn't have the `message_index`, is that okay?

Reply:
Cc @petyosi